docs/glossary.md

```diff
@@ -7,16 +7,16 @@ sidebar_label: Glossary
 ### All Aragon terminology
 ---
 
-- [**Aragon CLI**](/docs/cli-usage.html): Tool for creating, testing and publishing Aragon applications.
+- [**aragonCLI**](/docs/cli-usage.html): Tool for creating, testing and publishing Aragon applications.
 - [**aragonOS**](/docs/aragonos-intro.html): Framework that enables flexible and upgradeable governance mechanisms by creating and assigning permissions to multiple entities.
-- [**Aragon Package Manager**](/docs/apm.html): Decentralized package manager based on aragonOS that handles upgreadability of smart contracts and arbitrary data blobs, such as webapps.
-- **APM Repository**: Smart contract deployed inside APM that keeps track of the versions for a package.
-- [**aragon.js**](/docs/aragonjs-intro.html): Javascript library to interact with aragonOS by handling transaction pathing, upgradeability and state of the contracts.
-- [**Aragon UI**](/docs/aragonui-intro.html): Aragon-native toolkit of UI components for decentralized apps.
-- [**Aragon client**](http://app.aragon.org): Client to interact with aragonOS-powered decentralized organizations. It implements a signer with transaction pathing, notifications and a sandboxed environment for aragonOS-based apps using aragon.js.
+- [**aragonPM**](/docs/apm.html): Decentralized package manager based on aragonOS that handles upgreadability of smart contracts and arbitrary data blobs, such as webapps.
+- **aragonPM Repository**: Smart contract deployed inside aragonPM that keeps track of the versions for a package.
+- [**aragonAPI**](/docs/aragonjs-intro.html): Javascript library to interact with aragonOS by handling transaction pathing, upgradeability and state of the contracts.
```

Comment:
We need to reword this bit, as `aragonAPI` is an umbrella name for all user-facing API packages, and `@aragon/api` will be one particular implementation in javascript. This is also where the specs (#23) would live under.

How about:


- aragonAPI: Standard set of APIs and specifications used to interact with aragonOS-powered contracts by handling transaction pathing, upgradeability, and contract state. Reference implementations in specific languages include:
  - [JavaScript](/docs/aragonjs-intro.html)

The rest of the changes should make it clear that it's the JavaScript version of aragonAPI.